build/package.yml

```diff
@@ -21,4 +21,11 @@ steps:
     inputs:
       pathtoPublish: '$(build.artifactStagingDirectory)/nupkgs'
       artifactName: 'nuget'
-      publishLocation: 'container'
\ No newline at end of file
+      publishLocation: 'container'
+
+  - task: PublishSymbols@2
+    displayName: 'Publish Symbols'
+    inputs:
+      searchPattern: '$(build.artifactStagingDirectory)\**\*.pdb'
```

Comment:
`indexSources` needs to be `false`, right?

Reply:
`indexSources` defaults to true, and should be [per the documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/symbols?view=azure-devops#publish-symbols-to-azure-artifacts-symbol-server)

Reply:
But I think you want [this doc](https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/symbols?view=azure-devops#publish-portable-pdbs-to-azure-artifacts-symbol-server) as we're using SourceLink for indexing

Reply:
ok - updated the build task to match